compiler/src/dmd/dsymbolsem.d

```diff
@@ -544,7 +544,7 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             }
         }
         if ((dsym.storage_class & STC.auto_) && !inferred)
-            dsym.error("storage class `auto` has no effect if type is not inferred, did you mean `scope`?");
+            dsym.error("- storage class `auto` has no effect if type is not inferred, did you mean `scope`?");
```

Comment:
I used a colon first but it gets formatted with a space before the colon, so I think a hyphen looks better.